cpu/cc26xx-cc13xx/rf-core/ieee-mode.c

```diff
@@ -1101,6 +1101,8 @@ off(void)
 
   while(transmitting());
 
+  /* stopping the rx explicitly results in lower sleep-mode power usage */
+  rx_off();
```

Comment:
@bkozak-scanimetrics I'll be testing this over the weekend. Meanwhile, do we _really_ need this call here? Surely powering down analog in addition to the PD (which is what your changes to `rf_core_power_down()` should do under all circumstances) will result in the same current draw?